Add 3 new terms related to energy efficiency (#30)
billwuqin/draft-bclp-green-terminology: draft-bclp-green-terminology.md

: This control can span a network or a subset of it.

Energy Efficiency:

> : Refer to provide a useful output with as little energy as possible,

Reviewer: ```suggestion
: Refers to the concept of providing a useful output with as little energy as possible,
```

  are essential to demonstrating EE improvements.  An example of a common
  outcome-oriented metric is energy consumption per data volume or traffic unit.

Energy Efficiency Management:

> : Refer to a set of processes used to discover a inventory of capabilities,

Reviewer: ```suggestion
: Refers to a set of processes used to maintain an inventory of capabilities,
```

use specific metrics to measure, report and assess energy consumption of the network,
and control the use of available energy in an optimized manner in the

> network to ensure that the network and network-attached devices use energy in

Reviewer: ```suggestion
The overall goal is to ensure that the network and underlying devices use energy in
```

a resource conserved manner and at low cost for the nature of the

> application and the cost constraints of the organization while achieving the

Reviewer: ```suggestion
the services it provides and the cost constraints while achieving the
```

> network’s functional and performance requirements by improving overall network

Reviewer: ```suggestion
network’s functional and performance requirements (e.g.;, improving overall network
```
